src/algorithm/broad_phase/dbvt.rs

```diff
@@ -0,0 +1,68 @@
+//! DBVT broad phase collision test
+
+use std::cmp::Ordering;
+use std::fmt::Debug;
+
+use dbvt::{DiscreteVisitor, DynamicBoundingVolumeTree, TreeValue};
+use prelude::*;
+
+/// DBVT collision testing
+pub struct DbvtBroadPhase;
+
+impl DbvtBroadPhase {
+    /// Create a new DbvtBroadPhase, borrows the tree to check against
```

Comment:
Is doesn't really borrows the tree. Is doc string correct?

Reply:
Ah, no, the first version borrowed the tree. I'll fix later